drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -280,6 +280,8 @@ static int spi_mcux_configure(const struct device *dev,
 
 		SPI_SlaveTransferCreateHandle(base, &data->handle,
 					      spi_mcux_transfer_callback, data);
+
+		data->ctx.config = spi_cfg;
```

Comment:
Thank you for spotting this and for the fix. 
Can we delete the code in the Master `if` condition and move this to outside the `if` block so that it covers both Master and Slave cases.

Reply:
> Thank you for spotting this and for the fix.

No problem! Thanks for the review!

> Can we delete the code in the Master `if` condition and move this to outside the `if` block so that it covers both Master and Slave cases.

We could, however I think this fix is more robust to future master/slave configuration changes of these source files if kept this way.